lua/rip-substitute/config.lua

```diff
@@ -20,14 +20,23 @@ local defaultConfig = {
 	keymaps = {
 		-- normal & visual mode
 		confirm = "<CR>",
+		confirmAll = "<S-CR>",
 		abort = "q",
-		prevSubst = "<Up>",
-		nextSubst = "<Down>",
+		prevSubst = "<A-Up>",
+		nextSubst = "<A-Down>",
+		prevMatch = "<Up>",
+		nextMatch = "<Down>",
```

Comment:
same thing as with `confirm`, this one creates a breaking change for existing users, and should therefore be avoided.

I think for `prevMatch` and `nextMatch`, using `n` and `N` might actually be a good fit? It fits with how vim users expect the keys to work, it prevents the need of changing the `nextSubst` keymaps (and thus avoids a breaking change), and the default `n/N` are not really needed in the tiny popup window anyway.

Reply:
Done 👍